src/Solvers/Greedy.jl

```diff
@@ -53,31 +53,30 @@ function ssa_greedy_optimize(inputs, output, size, choose_fn=greedy_choose_simpl
     ssa_path = Vector{NTuple{2,Int}}()
 
     # step 1: deduplicate shapes by eagerly computing Hadamard products (i.e. element-wise multiplication)
-    remaining = Dict{Int,Set{Symbol}}(i => inds for (i, inds) ∈ enumerate(inputs))
+    remaining = Dict{Int,Set{Symbol}}(i => inds for (i, inds) ∈ enumerate(inputs)) # key -> ssa_id
```

Comment:
```suggestion
    remaining = Dict{Int,Set{Symbol}}(i => inds for (i, inds) ∈ enumerate(inputs))
```

Is this comment necessary? If so, put it like `key = ssa_id` because the `->` can be misleading.

Reply:
Sure, this comments is not really necessary.